src/templates/ObservabilityPackDetails.js

```diff
@@ -32,6 +36,22 @@ const ObservabilityPackDetails = ({ data, location }) => {
       packId: pack.id,
     }
   );
+
+  const getPackUrl = () => {
+    const baseUrl =
+      'https://github.com/newrelic/newrelic-observability-packs/tree/main/packs';
+
+    if (!pack.logoUrl) {
+      return baseUrl;
+    }
+
+    /**
+     * logoUrl looks something like: 'https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.8.2/packs/couchbase/logo.svg'
+     */
+    const [packName, ..._] = pack.logoUrl.split('/').slice(-2);
```

Comment:
Deconstruction allows you to just specify what you need, so if you just want the head, you only need to declare the `packName` variable.
```suggestion
    const [packName] = pack.logoUrl.split('/').slice(-2);
```

Reply:
Resolving this since @zstix already covered it in his comments